data_structures/linked_list/test_linked_list.py

```diff
@@ -111,25 +111,51 @@ def test_insertAfter_empty(empty_new_list):
 
 
 def test_insertAfter_multiple(new_ll_iter):
+    """test correct insertion after"""
     a = new_ll_iter
     a.insertAfter(2, 9)
     assert a.find(9) is True      
 
 
 def test_kth_raises_indexerror(new_ll_iter):
+    """handles error as intended"""
     a = new_ll_iter
     with pytest.raises(IndexError):
         a.kthFromEnd(6) 
 
 
 def test_kth_returns_one(new_ll_iter):
+    """returnse corrext value"""
     a = new_ll_iter.kthFromEnd(3)
     assert a.val == 1
 
 
 def test_kth_returns_first(empty_new_list):
+    """returns empty list"""
     a = empty_new_list
     a.insert(1)
     assert type(a.kthFromEnd(0)) is Node
 
-      
+
+def test_mergeLists_one(empty_new_list, new_ll_iter):
```

Comment:
Instead of numbering these tests, it is better practice to have what is being tested in the function name since that is what is displayed when `pytest` is run.